src/_configure_plextrac.sh

```diff
@@ -102,6 +104,22 @@ function setDefaultSecrets() {
   export IFS=$OLDIFS
 }
 
+function getCKEditorRTCConfig() {
+  # parses output and saves the result of the json meta data
+  # the last line, which only contains the JSON data, should be used
+  CKEDITOR_JSON=`compose_client exec plextracapi npm run ckeditor:environment:migration | grep '^{' || log "ERROR: Unable to run ckeditor:environment:migration"`
```

Comment:
Also you can check the presence of the script before invoking NPM via `--if-present`. I was able to test and get this working with something like:

```bash
if [ $(compose_client exec plextracapi npm run ckeditor:environment:migration --if-present) ]; then
    CKEDITOR_JSON=`compose_client exec plextracapi npm run ckeditor:environment:migration | grep '^{' || error "ERROR: Unable to run ckeditor:environment:migration"`
  else
    error "Can't find migration script for CKEditor"
    return 0
  fi
```